### ISSUE TITLE
Nova 5.x support

### ISSUE THREAD

**Issue by vbezruchkin** [ebess/advanced-nova-media-library#452]
Hi guys,

Thanks for your packages. Any chance to get this updated to support Nova 5.x released a few days ago?

Thanks

**Comment by vipulasa** [comment]
You can manually clone it, change the composer nova version to 5, and use it. Remember you need to change a single function definition in `advanced-nova-media-library/src/Fields/Media.php` line 290 to `public function resolve($resource, ?string $attribute = null): void`

**Comment by maul75** [comment]
I think this small adjustment should be incorporated into the base repo. Unless this repository is no longer maintained, then a clone / fork makes more sense. Maybe there will be a short update from the developers.

**Comment by daPhantom** [comment]
Any update on Nova 5 compatibility? This is the only blocking dependency we have as of right now.

**Comment by daPhantom** [comment]
> You can manually clone it, change the composer nova version to 5, and use it. Remember you need to change a single function definition in `advanced-nova-media-library/src/Fields/Media.php` line 290 to `public function resolve($resource, ?string $attribute = null): void`

Looks like this is not sufficient. I forked it and I do not have any controls on the thumbnails like deleting images, downloading, etc... So it's not 100% functional as of right now.

**Comment by gerardnll** [comment]
Correct. It needs changes on the JS side as @daPhantom says. Merged PR solves it partially.

**Comment by vbezruchkin** [comment]
Working fine with Nova 5.
